### PR TITLE
add warp test command for writing tests in solidity

### PR DESCRIPTION
**Pull request by BitBaseBit**

### CODE REVIEW

**Review by swapnilraj — APPROVED** [review]

File: README.md [hunk 1]
```diff
@@ -143,6 +145,12 @@ You can check the status of your transaction with:
 warp status TX_HASH
 ```
 
+## Testing with Warp
+
+You'll find an example of how to write tests in your solidity contract, and then call `warp test` to run them in `src/warp/test-tool/example`.
```
Comment (murcake) [inline README.md:150]: s/test-tool/test_tool/

File: src/warp/cli/cli.py [hunk 2]
```diff
@@ -165,5 +173,27 @@ def status(tx_hash, network):
     asyncio.run(_status(tx_hash, network))
 
 
+@warp.command()
+def test():
+    suffix = get_system_suffix(platform.system())
+    contracts_dir = os.path.join(os.getcwd(), "contracts")
+    tool_path = tool_path = (
+        Path(pkg_resources.get_distribution("sol-warp").location) / "warp" / "test_tool"
```
Comment (murcake) [inline src/warp/cli/cli.py:181]: `tool_path = tool_path = ` is excessive

**Review by murcake — APPROVED** [review]

**Review by murcake — APPROVED** [review]
Let's merge

File: src/warp/cli/cli.py (hunk 2 above)
Comment (murcake) [reply src/warp/cli/cli.py:181]: oh wait, it's not fixed yet

**Pull request merged**